src/App/App.css

```diff
@@ -1,5 +1,9 @@
-.home-header {
```

Comment:
Of note: might need to double check that our Cypress test is still passing for this.

Reply:
Good catch @abodnar1!